horizon_bsn/api/neutron.py

```diff
@@ -98,6 +132,10 @@ def reachabilitytest_create(request, **params):
     :param expected_result: expected result of the reachability test
     """
     LOG.debug("reachabilitytest_create(): params=%s", params)
```

Comment:
Do we need to change the log or skip it when it is not supported? Might not be important since it is debug level log.

Reply:
i haven't found the logfile or command to see this log. usually all upstream APIs have a debug log before doing anything. so i kept it.
shouldn't be an issue though.